js/charts.js

```diff
@@ -74,9 +74,86 @@ var charts = {
         }
       }
     });
-
   },
+  //Draws our Weights Chart
   'weight-sets-reps': (chartbox, historicalData) =>{
-    //TODO: Draw Chart for weight
-  }
-};
\ No newline at end of file
+    let weightArr = [];
+    let numOfRepsArr = [];
+    let numOfSetsArr = [];
+    let day = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
```

Comment:
Part of Matt's pull is bringing in dates. We should use those rather then assuming the second data set is Tuesday. It should be a few minutes before we can get that sorted (and applied) 

We're going to have to concider this blocked until we can pull that in